eoscostarica/gGoods
# Feat/free animals map

### Title

### What does this PR do?

- closes #78

### Steps to test

1. `make run`
2. Click 'Free your Animals' card in the game section and see the map

#### CheckList

- [] Follow proper Markdown format
- [] The content is adequate
- [] The content is available in both english and spanish
- [] I Ran a spell check
